_data/openzipkin_transport_integrations.yml

```diff
@@ -0,0 +1,50 @@
+- type: Collector
+  module: >-
+    [zipkin-aws collector-sqs](https://github.com/openzipkin/zipkin-aws/tree/master/autoconfigure/collector-sqs)
+  product: >-
+    [AWS SQS](https://aws.amazon.com/sqs/)
+  notes:
+
+- type: Collector
+  module: >-
+    [zipkin-aws collector-kinesis](https://github.com/openzipkin/zipkin-aws/tree/master/autoconfigure/collector-kinesis)
+  product: >-
+    [AWS Kinesis](https://aws.amazon.com/kinesis/)
+  notes:
+
+- type: Storage
+  module: >-
+    [zipkin-aws storage-elasticsearch-aws](https://github.com/openzipkin/zipkin-aws/tree/master/autoconfigure/storage-elasticsearch-aws)
+  product: >-
+    [AWS Elasticsearch Service](https://aws.amazon.com/elasticsearch-service/)
+  notes:
+
+- type: Storage
+  module: >-
+    [zipkin-aws storage-xray](https://github.com/openzipkin/zipkin-aws/tree/master/autoconfigure/storage-xray)
+  product: >-
+    [AWS X-Ray](https://aws.amazon.com/xray/)
+  notes: only supports sending to an XRay UDP daemon
+
+- type: Storage
+  module: >-
+    [zipkin-gcp storage-stackdriver](https://github.com/openzipkin/zipkin-gcp/tree/master/autoconfigure/storage-stackdriver)
+  product: >-
+    [GCP Stackdriver](https://cloud.google.com/stackdriver/)
+  notes: only supports sending to an Stackdriver
+
+- type: Collector
+  module: >-
+    [zipkin-collector kafka08](https://github.com/openzipkin/zipkin/tree/master/zipkin-autoconfigure/collector-kafka08)
+  product: >-
+    [Apache kafka](https://kafka.apache.org/)
+  notes: Kafka 8.2.2+
```

Comment:
Isn't it Kafka 0.8, not 8.2?

Reply:
good point https://github.com/openzipkin/openzipkin.github.io/commit/fa264020327dfe033b4f718c3a20e7352dbfbc67